backend/HoReD/Controllers/SalaryController.cs

```diff
@@ -30,7 +30,6 @@ public IHttpActionResult GetRatesForProfession(int professionId)
         [Route("api/Salary/Rate/delete")]
         public IHttpActionResult DeleteRate(Models.SalaryBindingModel model)
         {
-            //return Ok();
```

Comment:
Bad practice push commented code into server

Reply:
will be fixed